sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1559,6 +1559,22 @@ object RapidsConf {
     .integerConf
     .createWithDefault(value = 0)
 
+  val CONCURRENT_WRITER_PARTITION_FLUSH_SIZE =
+    conf("spark.rapids.sql.concurrentWriterPartitionFlushSize")
+        .doc("The flush size of the concurrent writer cache in bytes for each partition. " +
+            "If specified spark.sql.maxConcurrentOutputFileWriters, use concurrent writer to " +
+            "write data. Concurrent writer first caches data for each partition and begins to " +
+            "flush data if find one partition data is equal or greater than this size. " +
```

Comment:
nit: "flush the data if it finds one partition with a size that is greater than or equal to this config. "

Reply:
Done